unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -340,19 +340,29 @@ protected void HideAllAgentRenderers()
             HashSet<Vector3> seenPoints = new HashSet<Vector3>();
             int layerMask = 1 << 8;
             int stepsTaken = 0;
+
+            //Traverse entire room in every direction with every step, stopping at any points that repeat and continuing on from ones that don't yet
             while (pointsQueue.Count != 0) {
                 stepsTaken += 1;
                 Vector3 p = pointsQueue.Dequeue();
+                //Check if p is a point already reached from a different direction
                 if (!goodPoints.Contains(p)) {
+                    //Add p to HashSet of reachable positions
                     goodPoints.Add(p);
+                    //Initialize hashset ("list") of objects that agent is currently colliding with (but don't epxect it to have anything yet)
                     HashSet<Collider> objectsAlreadyColliding = new HashSet<Collider>(objectsCollidingWithAgent());
+                    //From p, move in each cardinal direction by d (gridSize * gridMultiplier)
```

Comment:
Another C# style convention, can you add a space above each comment, if it is not the start of a new indentation group? So this block goes from:
```c#
                Vector3 p = pointsQueue.Dequeue();
                //Check if p is a point already reached from a different direction
                if (!goodPoints.Contains(p)) {
                    //Add p to HashSet of reachable positions
                    goodPoints.Add(p);
                    //Initialize hashset ("list") of objects that agent is currently colliding with (but don't epxect it to have anything yet)
                    HashSet<Collider> objectsAlreadyColliding = new HashSet<Collider>(objectsCollidingWithAgent());
                    //From p, move in each cardinal direction by d (gridSize * gridMultiplier)
                    foreach (Vector3 d in directions) {
                        Vector3 newPosition = p + d * gridSize * gridMultiplier;
                        //If p + d is a point already reached from a different direction, move on to the next direction from p
                        if (seenPoints.Contains(newPosition)) {
                            continue;
                        }
```
to:
```c#
                Vector3 p = pointsQueue.Dequeue();

                // Check if p is a point already reached from a different direction
                if (!goodPoints.Contains(p)) {
                    // Add p to HashSet of reachable positions
                    goodPoints.Add(p);

                    // Initialize hashset ("list") of objects that agent is currently colliding with (but don't epxect it to have anything yet)
                    HashSet<Collider> objectsAlreadyColliding = new HashSet<Collider>(objectsCollidingWithAgent());

                    // From p, move in each cardinal direction by d (gridSize * gridMultiplier)
                    foreach (Vector3 d in directions) {
                        Vector3 newPosition = p + d * gridSize * gridMultiplier;

                        // If p + d is a point already reached from a different direction, move on to the next direction from p
                        if (seenPoints.Contains(newPosition)) {
                            continue;
                        }
```
Notice that `// Add p to HashSet of reachable positions` does not have a newline above it because it is starting a new indented further than the previousline.

Reply:
More to this than I realized. Done.